server/shared/src/main/scala/org/http4s/server/middleware/CORS.scala

```diff
@@ -178,6 +178,9 @@ object CORS {
   )
   val defaultVaryHeader: Header.Raw = Header.Raw(ci"Vary", "Origin,Access-Control-Request-Method")
 
+  private[this] final val accessControlAllowCredentialsHeaderTrue: Header.Raw =
+    Header.Raw(ci"Access-Control-Allow-Credentials", "true")
```

Comment:
This is kind of fighting the idea of having modeled headers in the first place.  Something like a `Access-Control-Allow-Credentials.true.toRaw` would achieve the same thing?

Line 179 is "cheating" in the same way, so it's not a new problem.

Reply:
Yeah I don't think much is changing here as you said: the codebase already supports both modelled and non-modelled headers. They're both used heavily in different locations in the code (see CORS header: it's a complete mix of both)